File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int lux = wallpaper.length - 1;
        int luy = wallpaper[0].length() - 1;
        int rdx = 0;
        int rdy = 0;
        
        for (int i = 0; i < wallpaper.length; i++) {
            for (int j = 0; j < wallpaper[i].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    lux = Math.min(lux, i);
                    luy = Math.min(luy, j);
                    rdx = Math.max(rdx, i + 1);
                    rdy = Math.max(rdy, j + 1);
                }
            }
        }
        
        int[] answer = {lux, luy, rdx, rdy};
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.kt
class Solution {
    fun solution(wallpaper: Array<String>): IntArray {
        var lux = wallpaper.size - 1
        var luy = wallpaper[0].length - 1
        var rdx = 0
        var rdy = 0
        
        for (i in wallpaper.indices) {
            for (j in wallpaper[i].indices) {
                if (wallpaper[i][j] == '#') {
                    lux = minOf(lux, i)
                    luy = minOf(luy, j)
                    rdx = maxOf(rdx, i + 1)
                    rdy = maxOf(rdy, j + 1)
                }
            }
        }
        
        return intArrayOf(lux, luy, rdx, rdy)
    }
}